0.4.2 Changelog and Release PR (#322)
bottlerocket-os/twoliter: CHANGELOG.md

## [Unreleased]

[unreleased]: https://github.com/bottlerocket-os/twoliter/compare/v0.4.2...HEAD

## [0.4.2] - 2024-07-11

> ### Added
> - Add support for crane family of tools for publishing and fetching kits ([#305], [#317]) 

Reviewer: ```suggestion
### Added

- Add support for crane family of tools for publishing and fetching kits ([#305], [#317]) 
```

### Changed

> - Update application inventory generation ([#310])

Reviewer: ```suggestion
- Update application inventory generation to respect kits ([#310])
```

- Share file descriptors to the build container to speed up directory I/O ([#302])
- Combine build and repack dockerfiles ([#302])
- Move updater wave default schedules into pubsys ([#321])
- Drop support for cgroup feature flags ([#318])

> - Add additional image feature flags ([#318])

Reviewer: Move to added

## [Unreleased]

[unreleased]: https://github.com/bottlerocket-os/twoliter/compare/v0.4.2...HEAD

## [0.4.2] - 2024-07-11

## Changed

> ## Changes

Reviewer 2: nit: This can be removed.

> - Updates to application inventory generations ([#310])

Reviewer: Generally I think the changelog should be in the imperative mood:

```suggestion
- Update application inventory generation ([#310])
```

This could also use some additional detail -- how was it updated?

Reviewer: This could maybe be something like `Update application inventory generation to accommodate kits`

> - Improvements for directory sharing during build ([#302])

Reviewer: ```suggestion
- Share file descriptors to the build container to speed up directory I/O ([#302])
```

- Combine build and repack dockerfiles ([#302])
- Move updater wave default schedules into pubsys ([#321])

> - Added support for crane family of tools for publishing kits ([#305]) 

Reviewer 2: These two lines could be moved to its own `Added` section. 

```suggestion
# Added

- Added support for crane family of tools for publishing kits ([#305]) 
- Added support for crane family of tools for fetching kits ([#317])

## Changed
...
```

> - Added support for crane family of tools for publishing kits ([#305]) 
> - Added support for crane family of tools for fetching kits ([#317])

Reviewer: ```suggestion
- Add support for crane family of tools for publishing and fetching kits ([#305], [#317])
```

- Drop support for cgroup feature flags (#[318])

> - Add additional image feature flags (#[318])

Reviewer: ```suggestion
- Drop support for cgroup feature flags ([#318])
- Add additional image feature flags ([#318])
```
